tests/playbooks/_stop_ping_test.yaml

```diff
@@ -0,0 +1,11 @@
+- name: Stop ping test
+  hosts: localhost
+  gather_facts: false
+  module_defaults:
+    ansible.builtin.shell:
+      executable: /bin/bash
+  tasks:
+    - name: stop the ping test and validate the result
+      when: prelaunch_test_instance|bool
+      ansible.builtin.shell: |
+        ~/stop_ping.sh 0 0
```

Comment:
I think we should parametrize the failure thresholds?

And maybe even parametrize whether we want to verify the ping test results at all? Essentially giving us a way to switch the ping test off if need be.

Reply:
The test_with_ceph and test_minimal playbooks are the basis of all our CI jobs. The jobs don't redefine this playbook, they call it. So by adding it to test_with_ceph.yaml we're in fact adding it to several jobs both upstream and downstream. I think that's good but i think we should have the flag :).

Also if we won't be stopping the ping test, we shouldn't be starting it either i think.

-----

Later we should add it to test_minimal.yaml too but for now i'd leave it untouched. In case something goes wrong with the pingtest in periodic/downstream jobs, at least the non-ceph ones will be unaffected for now.